_posts/2017-02-25-tickets-on-sale-and-timeline.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Supporter Tickets on Sale, prices and timeline"
+socialImageSrc: '/assets/posts/2017-tickets.png'
+authors:
+  - ben
+  - andrew
+---
+
+We hadn't fully announced the new website yet, and people already asked about when they can buy the tickets. Well, you are in luck, because as off today you can 
+
+[![Get your Ticket now (block of tickets with prices from website)](/assets/posts/2017-tickets.png)](http://2017.rustfest.eu/)
+
+You can find these on the main home page.
+
+### Timeline and prices
+
+The same as last year, we are starting with offering our [✨ Supporter ✨ Ticket](https://ti.to/asquera-event-ug/rustfest-kyiv/with/nhv7rcouqe0) first (from 199€+ _- available now_) and will sell Regular Tickets in batches again. In order to reward who is willing to commit early, the prices will increase with every batch. They will go on sale as follows:
```

Comment:
not a coupon, just the "official ticket url" (given from Ti.to). I think those are also used to link to specific hidden tickets but secondly, it highlights the Ticket in question and puts the focus in the right field, once there are more than just one ticket. I'd leave it in, doesn't hurt.